index.js

```diff
@@ -0,0 +1,209 @@
+const typeScriptSettings = {
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2018,
+    ecmaFeatures: {
+      jsx: true,
+    },
+    sourceType: 'module',
+    project: './tsconfig.json',
+  },
+  extends: [
+    'plugin:@typescript-eslint/recommended',
+    'prettier/@typescript-eslint',
+    'plugin:prettier/recommended',
+  ],
+  rules: {
```

Comment:
Got an error in a `d.ts` file, seems we need to add:
```
    'no-useless-constructor': 'off',
    '@typescript-eslint/no-useless-constructor': 'error',
```

Otherwise empty constructors (will only exist in TS) will break the linting with a runtime error.